src/plugins/about/index.js

```diff
@@ -0,0 +1,133 @@
+const log = require('Debug')('log:about');
```

Comment:
Causes cockpit to fail fatally in same way as: https://github.com/OpenROV/openrov-software/issues/573